tests/e2e_test.go

```diff
@@ -39,8 +40,16 @@ var (
 
 	decider  *exec.Cmd
 	cancelFn context.CancelFunc
+
+	e2eFlags *e2e.FlagVars
 )
 
+func TestMain(m *testing.M) {
+	e2eFlags = e2e.RegisterFlags()
+	flag.Parse()
+	os.Exit(m.Run())
+}
+
```

Comment:
Is this used somewhere/somehow?

Reply:
Yes -- I do believe this is necessary to be able to pass custom flags such as `--activate-granite` to ginkgo test run.

Reply:
Oh interesting, yeah I see how it's used in subnet-evm now. I had the directory excluded from my searches